src/libcprover-cpp/api_options.cpp

```diff
@@ -2,52 +2,71 @@
 
 #include "api_options.h"
 
-#include <util/cmdline.h>
 #include <util/make_unique.h>
-#include <util/options.h>
 
-#include <ansi-c/goto_check_c.h>
-#include <goto-checker/solver_factory.h>
+struct api_optionst::implementationt
+{
+  // Options for the verification engine
+  bool simplify_enabled;
+
+  // Option for dropping unused function
+  bool drop_unused_functions_enabled;
+
+  // Option for validating the goto model
+  bool validate_goto_model_enabled;
```

Comment:
I recognise this is a refactor, but it would be nice if these comments were more consistent. Maybe actually describe what the option does and have their name align be meaningful.

I notice these have better comments and names in other places.